gradle/libs.versions.toml

```diff
@@ -74,22 +62,23 @@ netty-tcnative-classes = { module = "io.netty:netty-tcnative-classes", version.r
 netty-transport-epoll = { module = "io.netty:netty-transport-native-epoll", version.ref = "netty" }
 netty-unix-common = { module = "io.netty:netty-transport-native-unix-common", version.ref = "netty" }
 okhttp = "com.squareup.okhttp:okhttp:2.7.5"
-okio = "com.squareup.okio:okio:2.10.0"
+# okio 3.5+ uses Kotlin 1.9+ which requires Android Gradle Plugin 9+
+okio = "com.squareup.okio:okio:3.4.0"
```

Comment:
For posterity, last time the update to 3.4.0 failed with this error: 

> `Execution failed for task ':grpc-all:checkUpperBoundDeps'.
Maven version skew: com.squareup.okio:okio (3.4.0 != null) Bad version dependency path: [project ':grpc-all', io.grpc:grpc-okhttp:1.58.0-SNAPSHOT] Run './gradlew :grpc-all:dependencies --configuration runtimeClasspath' to diagnose`
> — https://github.com/grpc/grpc-java/pull/10359#issuecomment-1632853307

I guess they fixed it without re-release? Or it somehow got resolved on our side.

Reply:
https://github.com/grpc/grpc-java/commit/b526a68622db3d202dd66e5fdb5fa510ecc09084 probably fixed it.

Reply:
IRRC I rebased my branch against master with this commit: https://github.com/grpc/grpc-java/blob/419767f96811168afe36f39c94cf7b7be4441042/build.gradle#L512C44-L517